keel-sql/src/main/resources/db/databaseChangeLog.yml

```diff
@@ -32,9 +32,6 @@ databaseChangeLog:
   - include:
       file: changelog/20190904-rename-name-to-id.yml
       relativeToChangelogFile: true
-  - include:
-      file: changelog/20190917-rename-cluster-to-server-group.yml
-      relativeToChangelogFile: true
```

Comment:
I removed this previous migration as it no longer serves any purpose. Given the change in structure between `ServerGroupSpec` and `ClusterSpec` I don't think there's any point in trying to migrate existing resources, we should just re-create them.